src/server/v2.0/handler/artifact.go

```diff
@@ -98,14 +101,17 @@ func (a *artifactAPI) GetArtifact(ctx context.Context, params operation.GetArtif
 	}
 	// set option
 	option := option(params.WithTag, params.WithImmutableStatus,
-		params.WithLabel, params.WithScanOverview, params.WithSignature)
+		params.WithLabel, params.WithSignature)
 
 	// get the artifact
 	artifact, err := a.artCtl.GetByReference(ctx, fmt.Sprintf("%s/%s", params.ProjectName, params.RepositoryName), params.Reference, option)
 	if err != nil {
 		return a.SendError(ctx, err)
 	}
-	return operation.NewGetArtifactOK().WithPayload(artifact.ToSwagger())
+	art := &model.Artifact{}
+	art.Artifact = *artifact
+	a.assembleArtifact(ctx, art, params.WithScanOverview)
```

Comment:
Maybe we can provide a utility method to handle scan summary and possible signature etc.

`ContentBuilder.Traget(*Arrtifact).WithScanSummary().WithSignature().WithLabels().FillIn()`